farmerbot/cmd/run.go

```diff
@@ -40,6 +41,15 @@ var runCmd = &cobra.Command{
 		if err != nil {
 			return err
 		}
+		sub, err := substrate.NewManager(internal.SubstrateURLs[network]...).Substrate()
+		if err != nil {
+			return err
+		}
+		defer sub.Close()
```

Comment:
no connection was opened here so no need to close the manager

Reply:
Substrate( ) by default returns a connection :
![image](https://github.com/user-attachments/assets/8d8246b3-f1b2-4d74-b1d4-3ad531a03a12)
doesn't it make connection to db?

Reply:
ahaaa, that's kinda confusing, please create a manager then a connection, or at least rename to subConn, I couldn't see your're using a connection